transmit-lib/src/cmr/transmit/search.clj

```diff
@@ -89,6 +89,27 @@
       (errors/internal-error!
         (format "Granule search failed. status: %s body: %s" status body)))))
 
+(defn-timed find-granule-references-return-errors
```

Comment:
When I recommended this current name, I wasn't expecting you to change this function in a way that it does not function with valid searches. Since you have modified the function in a way that it does not return any result when the search is valid, we should rename this function to something like `validate-granule-search-params`.